pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/release-notes/index.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+title: Release Notes
+menuWeight: 40
+excerpt: New features, updates, and known limitations
+render: mustache
+model: ../data.yml
+---
+
```

Comment:
Must have a date for release.

Reply:
Can you link an example?

Reply:
https://github.com/mesosphere/dcos-docs-site/pull/3113/files#diff-62e96a3337cf3ddceb18058a74c7e938R11